App/automotive/src/main/res/layout/fragment_app_settings.xml

```diff
@@ -13,6 +13,7 @@
             android:layout_height="wrap_content">
 
             <include
+                android:id="@+id/theme_mode_tile"
```

Comment:
ich fänds gut wenn das ne andere id in der view bekommen würde, da wir das vorherige tile ja behalten wollen

Reply:
Hab die Ids jetzt umbennant zu @+id/vehicle_theme_mode_tile" und @+id/app_theme_mode_tile"